hax/helper/generate_sysconf.py

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+# Module for Hare to create motr and s3server environment files.
+
+from typing import Dict, List, NamedTuple, Optional
+from hax.util import repeat_if_fails, KVAdapter
+from hax.types import Fid, ObjT
+import logging
+import os
+import re
+from helper.exec import Executor, Program
+import simplejson
+import fileinput
+
+
+def get_node_name():
+    with open('/var/lib/hare/node-name') as f:
```

Comment:
/var/lib/hare/node-name is created by `configure` script. We need to be sure that when generate_sysconf.py is invoked this file is present. I think we better take node-name as a input to the respective functions.

Reply:
took as a parameter. done